Veil-light-wallet-Linux.html

```diff
@@ -0,0 +1,95 @@
+        <ul>For Linux, look for the appropriate release for Ubuntu.</ul>
+        <figure>
+            <img src="light-wallet-pictures/veil_wallet_linux.png" alt="veil_wallet_linux_photo">
+            <figcaption>Select the Linux version.</figcaption>
+        </figure>
+        <li>Download the package suitable for Ubuntu.</li>
+    </ol>
+
+    <h2>Installing the Veil Light Wallet:</h2>
+    <ol>
+        <li>Unzip the package to a location you know. For this example I'm saving it to my desktop for simplicity.</li>
+        <li>If the files are not in a folder, create a new folder and drag the files into it.</li>
+        <figure>
+            <img src="light-wallet-pictures/new_folder_w_files.png" alt="new_folder_w_files">
+            <figcaption>New folder on top.<br>
+            Downloaded wallet files on bottom.</figcaption>
+        </figure>
+        <li>Open the terminal and navigate to your wallet location. If you have the folder on your desktop, <br>
+            type <code>cd Desktop</code> and then <code>cd veil\ light\ wallet/</code> (or whatever you named the folder).</li>
+            <figure>
+                <img src="light-wallet-pictures/light_wallet_example2.png" alt="light_wallet_example">
+            </figure>
+        <li>Once in the folder, type <code>chmod +x veil-wallet</code> to make the veil-wallet file executable.</li>
+            <figure>
+                <img src="light-wallet-pictures/chmod_example.png" alt="chmod_example"> 
+            </figure>
+    </ol>
+
+    <h2>Launching Veil Light Wallet:</h2>
+    <ol>
+        <li>To start the wallet in the terminal, use <code>./veil-wallet</code>.</li>
+            <figure>
+                <img src="light-wallet-pictures/cd_Desktop.png" alt="cd_Desktop">
+                <figcaption>Wallet start example.</figcaption>
+            </figure>
+        <li>Alternatively, close the terminal and double-click the veil-wallet gear icon in your veil light wallet folder.</li>
+            <figure>
+                <img src="light-wallet-pictures/veil_wallet_gear_icon.png" alt="veil_wallet_gear_icon">
+                <figcaption>Veil-wallet gear icon.</figcaption>
+            </figure>
+        <li>This should successfully launch your Veil Light Wallet.</li>
+        <li>Configure any initial settings, such as language.</li>
+        <li>Ensure you record and securely store your seed phrase generated during the setup. You can also enter a seed phrase from a previous Veil wallet.</li>
+        <li>After entering your wallet phrase correctly, let your wallet load.</li>
+        <li>The wallet should be ready to use in a few seconds up to a half-hour.</li>
+        <li>Note that since Veil is private by default, this light wallet uses RingCT, and basecoin will not be usable, visible, or available with this light wallet.</li>
```

Comment:
Same as for iphone guide (CT is also currently unavailable)

Reply:
Will add that CT and basecoin are unavailable.